src/platforms/common/performance/sampling.mdx

```diff
@@ -49,7 +50,7 @@ Whatever a transaction's sampling decision, that decision will be passed to its
 
 If the transaction currently being created is one of those subsequent transactions (in other words, if it has a parent transaction), the upstream (parent) sampling decision will always be included in the sampling context data, so that your <PlatformIdentifier name="traces-sampler" /> can choose whether and when to inherit that decision. (In most cases, inheritance is the right choice, so that you don't end up with partial traces.)
 
-For convenience, the <PlatformIdentifier name="traces-sampler" /> function can return a boolean, so that a parent's decision can be returned directly if that's the desired behavior.
+In some SDKs, for convenience, the <PlatformIdentifier name="traces-sampler" /> function can return a boolean, so that a parent's decision can be returned directly if that's the desired behavior.
```

Comment:
Is this something we diverged? Should we align (one way or the other?)

Reply:
This I believe was written with Python and JS in mind, where a function can return different data types. In Java to achieve the same we would have to return a wrapper object. I don't think it's worth it..?